src/Makefile

```diff
@@ -26,7 +26,7 @@ GIT_SOURCE        ?= https://github.com/greenpeace/planet4-base
 
 # The branch to checkout of GIT_SOURCE, eg:
 # Use local branch name if not set
-GIT_REF           ?= main
+GIT_REF           ?= artifact
```

Comment:
I'll revert that before merge. This is here to use the relevant branch from the base repository, which in turn uses app repos where the Github Archive plugin [has been removed](https://github.com/greenpeace/planet4-master-theme/commit/2559b50b85c6bf750a0790ff9825b846558080c7).